src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -452,7 +452,7 @@ int RadosBucket::remove_bucket(const DoutPrefixProvider* dpp,
 
   // remove lifecycle config, if any (XXX note could be made generic)
   (void) store->getRados()->get_lc()->remove_bucket_config(
-    this, get_attrs());
+    this, get_attrs(), false);
```

Comment:
what does the false do?  maybe an inline comment there?

Reply:
thanks, i added a constexpr variable to name that argument, and added a comment saying why